ui/app/src/models/GlobalState.ts

```diff
@@ -92,7 +92,7 @@ export interface GuestData {
   modelId: string;
   path: string;
   selected: EditSelection[];
-  itemBeingDragged: boolean;
+  itemBeingDragged: number;
```

Comment:
What changed here?

Reply:
Nothing changed, the type was wrong since when setting up `itemBeingDragged` the iceId is the value used in the payload, iceId being a number